build/transpile.js

```diff
@@ -960,7 +960,12 @@ class Transpiler {
 
         let bodyAsString = body.join ("\n")
 
+        const isAlias = bodyAsString.match ("'alias': true")
+        
         let header = this.createPHPClassHeader (className, baseClass, bodyAsString, async ? 'ccxt\\async' : 'ccxt')
+        if (isAlias && async) {
```

Comment:
@pcriadoperez I don't fully understand the issue yet, for instance https://github.com/ccxt/ccxt/blob/master/php/pro/okex.php seems ok or am I missing something?

Reply:
So strange seems to be working correctly now.. maybe it was an issue of a branch I was on.🤷‍♀️